### PR TITLE
Auto updater finishing touches

### PR DESCRIPTION
**Pull request by dpgraham**
* Simplified the logic for the autoUpdater. AutoUpdater on main process manages it's own state and then synchronizes the state with the BrowserWindow
* Added CSS styling

### CODE REVIEW

**Comment by dpgraham** [comment]
@jlipps

**Comment by dpgraham** [comment]
Made the browser window not resizable; kept progress bar when download complete

File: app/renderer/components/Updater/CheckForUpdates.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import {  } from 'antd';
```
Comment (jlipps) [inline app/renderer/components/Updater/CheckForUpdates.js]: if we're not importing anything, let's not import anything :-)

**Comment by dpgraham** [comment]
My master branch is ahead of this pull request so I'm just going to include the changes in a future PR.

**Comment by jlipps** [comment]
remember to squash next time to keep git history clean :-)

**Pull request merged**